src/client/utils.test.ts

```diff
@@ -1,6 +1,12 @@
+import { HttpResponse, http } from 'msw'
```

Comment:
I updated the test to use `msw` instead of running the actual server. This is the same way as `src/client/client.test.ts`.